### DISCUSSION TITLE
Skill Settings

### DISCUSSION THREAD

**Discussion by dcveteran** [OpenVoiceOS/ovos-buildroot#195] [Q&A] [answered]
Has anyone been successful in modifying settings in the settingsmeta.json file for an installed skill? I am attempting to change the weather units to fahrenheit. These are the default settings:

```
"name": "units",
"type": "select",
"label": "Temperature units",
"options": "Default (from Basic Settings)|default;Celsius|celsius;Fahrenheit|fahrenheit",
"value": "default"
```
I tried changing value to fahrenheit, "fahrenheit", and "Fahrenheit" but none of those have worked. I am making changes according to this document: https://openvoiceos.github.io/ovos-technical-manual/skill_settings/ ...but it seems I am missing something. Does the old mycroft.conf file need to be modified?

**Comment by builderjer** [comment]
You can change your units in the main config file. 

Create or edit the file at `~/.config/mycroft/mycroft.conf`

Add
```
{
  "system_units": "imperial"
}
```
For Fahrenheit and "metric" for Celsius

**Reply by dcveteran** [reply]
Thank you for the suggestion, but it did not work. That same directory has a web_cache.json file that holds what I think is the same data the old mycroft.conf file had--location, timezone, etc. (though I could be remembering incorrectly). I tried adding the "system_units" line to it as well but the OS reverts the file after I save it. For some reason it is defaulting to Lawrence, KS as the location (I am on the East Coast). 

Oh, I am running the Buildroot image on a Pi 4 since I have the Raspberry Pi screen attached. 

Any other suggestions? I'll keep poking around and see what I figure out. My goal is to update the documentation as I go.

**Reply by builderjer** [reply, marked as answer]
The `web_cache` file is applied before `~/.config/mycroft/mycroft.conf`

And if using a personal backend, that is where the settings are uploaded to. When using `no backend`, the default, it provides default data. So yes, it is expected to be overwritten.

I mistyped the setting

`"system_unit": "imperial"`
https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/mycroft.conf#L31

**Reply by dcveteran** [reply]
Thank you again, and thank you for the link--that is what I needed. I tried multiple variations of the imperial setting after your first reply (plural and singular) with no changes. After reviewing the formatting of your linked file, I see that the comma was the missing component. There was also a missing comma in the default mycroft.conf on my system which may have also been part of the problem. Once those commas were corrected, everything fell in line.

**Reply by builderjer** [reply]
you can check your config file with the command `cat ~/.config/mycroft/mycroft.conf | jq`.  If there are no errors, it will just cat the file, otherwise, it will throw an error. and it is usually a comma or bracket missing

**Comment by geetee24** [comment]
I tried that and rebooted (no errors in json) and suddenly my /dev/root is 100% full when it is just a rpi ovos standard image. How find the error files
